File: Exercises - LeetCode/1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        boolean stop = false;
        int[] res = new int[2];
        
        for (int i = 0; i < nums.length && !stop; i++) {
            for (int j = 0; j < nums.length && !stop; j++) {
                if (j != i && nums[i] + nums[j] == target ) {
                    stop = true;
                    res[0] = i;
                    res[1] = j;
                }
            }
        }
        return res;
    }
}

File: Exercises - LeetCode/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res;
        for(int i = 0; i < nums.size(); i++){
            int num = nums[i];
            for(int j = i + 1; j < nums.size(); j++) {
                if(num + nums[j] == target) {
                    return vector<int>{i, j};
                }
            }
        }
        return vector<int>{};
    }

};
